asset/js/library-menu.js

```
@@ -0,0 +1,220 @@
+
+    /**
+     * Handle keyboard navigation in menu
+     */
+    $(document).on('keydown', '.top-level-page-link', function(e) {
+        var $links = $('.top-level-page-link');
+        var currentIndex = $links.index(this);
+        
+        switch(e.key) {
+            case 'ArrowDown':
+                e.preventDefault();
+                var nextIndex = (currentIndex + 1) % $links.length;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)